### PR TITLE
Modified temporal projection roles. Wubba temporal projection is now …

### PR DESCRIPTION
**Pull request by EvidentlyCube**
…resistant to body attacks + Power token now gives abilities to temporal projections that are already active (there was a bug with stalwart role, which would inherit shallow water traversal from player IF player touched power token before recording, but would not do so if the token was activated during playback)

[Relevant thread](http://forum.caravelgames.com/viewtopic.php?TopicID=40546)

### CODE REVIEW

File: DRODLib/Slayer.cpp [hunk 1]
```diff
@@ -111,6 +112,7 @@ void CSlayer::Process(
 	if (!GetTarget(this->wTX, this->wTY, false))
 		return;
 
+	const CMonster* pTarget = this->pCurrentGame->pRoom->GetMonsterAtSquare(this->wTX, this->wTY);
```
Comment (mrimer) [inline DRODLib/Slayer.cpp:115]: This line appears unnecessary.  Remove.

**Review by mrimer — APPROVED** [review]
Thank you!
I noted one line to remove.

File: DRODLib/DbRooms.cpp [hunk 2]
```diff
@@ -4088,6 +4088,8 @@ void CDbRoom::ActivateToken(
 					{
 						CTemporalClone *pClone = DYN_CAST(CTemporalClone*, CMonster*, pMonster);
 						pClone->bIsTarget = true;
+						pClone->SetMovementType();
+
```
Comment (mrimer) [inline DRODLib/DbRooms.cpp:4092]: Remove blank line.

**Comment by EvidentlyCube** [comment]
@mrimer Pushed fixes!

**Review by mrimer — APPROVED** [review]

**Pull request merged**